test/config/config.go

```diff
@@ -187,6 +189,11 @@ func GetJobsServiceImageTag() string {
 	return env.jobsServiceImageTag
 }
 
+// GetConfigManagementConsoleImageTag return the Kogito Management Console image tag
+func GetConfigManagementConsoleImageTag() string {
```

Comment:
the `Config` in `GetConfigManagementConsoleImageTag` can now be removed as we moved the config to irs own package

Reply:
Fixed.